python/ray/train/v2/api/data_parallel_trainer.py

```diff
@@ -160,12 +160,21 @@ def _initialize_and_run_controller(self, **controller_init_kwargs) -> Result:
             controller.run()
             return controller.get_result()
 
-    @Deprecated
     @classmethod
+    @Deprecated
```

Comment:
Just curious does this change the behavior? Or just the readability?

Reply:
The original ordering actually doesn't work for some reason. Deprecated must wrap the regular method first before the classmethod wraps it...